launchpod/angular-launchpod/backend/src/test/java/com/google/launchpod/FormHandlerServletTest.java

```diff
@@ -172,97 +177,106 @@ public void doPost_ReturnsCorrectUrl() throws IOException {
     assertEquals(1, ds.prepare(new Query(USER_FEED)).countEntities(withLimit(10)));
 
     Query query = new Query(USER_FEED);
-    Entity entity = ds.prepare(query).asSingleEntity();
+    PreparedQuery preparedQuery = ds.prepare(query);
+    Entity desiredEntity = preparedQuery.asSingleEntity();
+
+    String expectedXmlString = RSS.toXmlString(TEST_RSS_FEED);
+    assertEquals(expectedXmlString, desiredEntity.getProperty(XML_STRING).toString());
 
-    assertEquals(TEST_PODCAST_TITLE, entity.getProperty(PODCAST_TITLE).toString());
-    assertEquals(TEST_MP3_LINK, entity.getProperty(MP3_LINK).toString());
-    RSS rssFeed = new RSS(TEST_PODCAST_TITLE, TEST_MP3_LINK);
-    TEST_XML_STRING = RSS.toXmlString(rssFeed);
-    assertEquals(TEST_XML_STRING, entity.getProperty(XML_STRING).toString());
+    String test_xml_string = RSS.toXmlString(TEST_RSS_FEED);
+    assertEquals(test_xml_string, desiredEntity.getProperty(XML_STRING).toString());
 
-    String id = Long.toString(entity.getKey().getId());
+    String id = KeyFactory.keyToString(desiredEntity.getKey());
     String rssLink = BASE_URL + id;
+    System.out.println("THIS IS THE EXPECTED LINK: " + rssLink);
+    System.out.println("THIS IS THE ACTUAL   LINK: " + stringWriter.toString());
+    System.out.println("length 1=" + rssLink.length());
+    System.out.println("length 2=" + stringWriter.toString().length());
```

Comment:
this should be changed to an assertEquals. expected value and actual value will be printed by assertEquals if a comparison fails